src/locales/it/translations/actions/index.ts

```diff
@@ -0,0 +1,15 @@
+export default {
+  actions: {
+    label: 'Azioni',
+    edit: 'Modifica',
+    save: 'Salva',
+    update: 'Aggiorna',
+    complete: 'Completo',
+    delete: 'Rimuovi',
+    cancel: 'Cancella',
+    new: 'Nuovo',
+    list: 'Lista',
+    search: 'Cerca',
+    confirmDelete: 'Cancella conferma',
```

Comment:
It would be more accurate to translate it into "Conferma cancellazione" or "Conferma eliminazione"

Reply:
You are totally right, why I wrote 'Cancella conferma'.... doesn't make sense